lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -586,6 +586,15 @@ module.exports = class MethodMappings {
         async clearElementValue(webElementOrId) {
           const element = this.getWebElement(webElementOrId);
           await element.clear();
+          try {
+            const value = await element.getProperty('value');
+            if (value.length !== 0 && isString(value)) {
```

Comment:
If the `value` is string should be checked before checking the length, otherwise the statement would throw error if the `value` is `null` or `undefined`.

```suggestion
            if (isString(value) && value.length > 0) {
```